docs/reference/dependencies.md

```diff
@@ -10,7 +10,7 @@ A complete list of all dependencies and third-party libraries for all the compon
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-<Tabs groupId="dependencies" defaultValue="zeebe" values={
+<Tabs groupId="dependencies" defaultValue="zeebe" queryString values={
```

Comment:
@pepopowitz this was almost _too_ easy.

I did notice because we have a trailing slash in our docs the queryString that's added ends up looking like `https://docs.camunda.io/docs/next/reference/dependencies/?dependencies=zeebe` which is fine, but looks a bit silly. I'm just excited for deep linking!

Reply:
yeah, that trailing slash thing has always been annoying to me since the first time I generated a static site, but I think it's the price you pay when every friendly markdown filename turns into a nicely named folder with an index.html.